WordPress/src/main/java/org/wordpress/android/WordPressDB.java

```diff
@@ -1776,25 +1798,23 @@ public boolean saveTheme(Theme theme) {
         boolean returnValue = false;
 
         ContentValues values = new ContentValues();
-        values.put("themeId", theme.getThemeId());
-        values.put("name", theme.getName());
-        values.put("description", theme.getDescription());
-        values.put("screenshotURL", theme.getScreenshotURL());
-        values.put("trendingRank", theme.getTrendingRank());
-        values.put("popularityRank", theme.getPopularityRank());
-        values.put("launchDate", theme.getLaunchDateMs());
-        values.put("previewURL", theme.getPreviewURL());
-        values.put("blogId", theme.getBlogId());
-        values.put("isCurrent", theme.isCurrent());
-        values.put("isPremium", theme.isPremium());
-        values.put("features", theme.getFeatures());
+        values.put(Theme.ID, theme.getId());
+        values.put(Theme.AUTHOR, theme.getAuthor());
+        values.put(Theme.SCREENSHOT, theme.getScreenshot());
+        values.put(Theme.AUTHOR_URI, theme.getAuthorURI());
+        values.put(Theme.DEMO_URI, theme.getDemoURI());
+        values.put(Theme.NAME, theme.getName());
+        values.put(Theme.STYLESHEET, theme.getStylesheet());
+        values.put(Theme.PRICE, theme.getPrice());
+        values.put(Theme.BLOG_ID, theme.getBlogId());
+        values.put(Theme.IS_CURRENT, theme.getIsCurrent() ? 1 : 0);
 
         synchronized (this) {
             int result = db.update(
                     THEMES_TABLE,
                     values,
-                    "themeId=?",
-                    new String[]{ theme.getThemeId() });
+                    Theme.ID + "=?",
+                    new String[]{ theme.getId() });
```

Comment:
nitpick; odd that you add spaces in the array initializer list here but remove them above.

ie `new String[]{blogId, "uploading"}`